benchmarks/viscoelastic_stress_build-up/viscoelastic_stress_build-up_particles.prm

```diff
@@ -0,0 +1,203 @@
+# This parameter file reproduces a 2D analytical benchmark for the build-up
```

Comment:
This looks a lot like a modified copy of the `viscoelastic_stress_build-up.prm` file in that folder. Could you use the `include` directive (as in `tests/rigid_shear_instantaneous.prm`) and then only keep the modified commands in this file? That would also make it simpler to keep up to date.